presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/EnvironmentUp.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.product.launcher.cli;
+
+import com.github.dockerjava.api.DockerClient;
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Module;
+import io.airlift.airline.Command;
+import io.airlift.log.Logger;
+import io.prestosql.tests.product.launcher.Extensions;
+import io.prestosql.tests.product.launcher.LauncherModule;
+import io.prestosql.tests.product.launcher.env.Environment;
+import io.prestosql.tests.product.launcher.env.EnvironmentModule;
+import io.prestosql.tests.product.launcher.env.EnvironmentOptions;
+import io.prestosql.tests.product.launcher.env.Environments;
+import io.prestosql.tests.product.launcher.env.SelectedEnvironmentProvider;
+import org.testcontainers.DockerClientFactory;
+
+import javax.inject.Inject;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+
+import static io.prestosql.tests.product.launcher.cli.Commands.runCommand;
+import static io.prestosql.tests.product.launcher.testcontainers.TestcontainersUtil.killContainersReaperContainer;
+import static java.util.Objects.requireNonNull;
+
+@Command(name = "up", description = "start an environment")
+public final class EnvironmentUp
+        implements Runnable
+{
+    private static final Logger log = Logger.get(EnvironmentUp.class);
+
+    @Inject
```

Comment:
How does it work? Where do inject? Into `EnvironmentUp` or `EnvironmentOptions`?

Reply:
this is standard thing in https://github.com/airlift/airline